markdown/README

```diff
@@ -155,12 +155,12 @@ License
 The Markdown plugin is licensed under the GNU General Public License,
 version 2. For the full text of the license, please visit
 http://www.gnu.org/licenses/gpl-2.0.html. The GPL license covers all code
-and files *not* in the ``discount`` directory.
+and files *not* in the ``peg-markdown/peg-0.1.9`` directory.
 
 All code inside the ``peg-markdown/peg-0.1.9`` directory is under the MIT
 license (see the ``peg-markdown/LICENSE`` file) and all contributions to
 this code *will* remain under this license. This will make it easier to
-integrate improvements to Discount back upstream if it ever makes sense.
+integrate improvements to Peg back upstream if it ever makes sense.
```

Comment:
Not sure if it matters much, but IIUC peg-markdown itself is dual-licensed `GPL-2.0-or-later OR MIT`, so it'd make sense to keep all modifications under this for best upstreamability -- although upstream seems dead.